hotness/buildsys.py

```diff
@@ -124,7 +118,19 @@ def handle(self, package, upstream, version, rhbz):
 
             session = self.session_maker()
             task_id = self.scratch_build(session, package, srpm)
-            return task_id
+
+            # Now, craft a patch to attach to the ticket
+            output = self.run(['git', 'commit', '-a',
+                               '-m', comment,
+                               '--author', self.userstring], cwd=tmp)
+            filename = self.run(['git', 'format-patch', 'HEAD^'], cwd=tmp)
+            filename = filename.strip()
+
+            # Copy the patch out of this doomed dir so bz can find it
+            destination = '/var/tmp/' + filename
+            shutil.move('/'.join([tmp, filename]), destination)
```

Comment:
This line scares me a little as it would accept a folder as well as a file, bu on the other side things are pretty contained about what they can do.

Btw, ``'/'.join()`` not ``os.path.join()``?